test/functional/test_vm_default.py

```diff
@@ -6,7 +6,7 @@
     vm status: running
     vm cpu count: 4
     vm disk count: 1
-    vm disk size: 16G
+    vm disk size: 4G
```

Comment:
Default is Quanta D51 correct? And by default its disk capacity is changed to 4G?